### PR TITLE
Add plugin support and a plugin to get menegement-admin secret 

### PR DESCRIPTION
**Pull request by moolitayer**
Enables running arbitrary scripts on the master node.
Includes a script to output management-infra secret.

### CODE REVIEW

**Comment by moolitayer** [comment]
Tested in http://cm-jenkins.usersys.redhat.com/job/ocp-ansible-jenkins-testing/2/console

**Comment by moolitayer** [comment]
cc @ilackarms @zgalor @simon3z 

This allows running arbitrary scripts on the master node.
We can use to do any of:
- run Ansible automation (add ManageIQ provider with prometheus endpoints, add policies etc)
- Install podified ManageIQ (at least until we can do it properly with Ansible as part of our main deployment)

**Review by ilackarms — APPROVED** [review]
very cool, I like it

**Comment by simon3z** [comment]
@moolitayer @ilackarms it would be nice to have also the CA printed out so we'll make it easier to always use SSL :blush:

File: deployer-test.sh [hunk 1]
```diff
@@ -16,6 +16,7 @@ LDAP_PROVIDERS="[{'name': 'test', 'challenge': 'true', 'login': 'true', 'kind':
 MASTER_IP=10.35.48.200
 OPENSHIFT_ANSIBLE_REF=openshift-ansible-3.6.140-1
 OPENSHIFT_ANSIBLE_REPO_URL=https://github.com/openshift/openshift-ansible
+PLUGIN_URLS="https://github.com/moolitayer/ocp-ansible-jenkins/blob/master/plugins/osh_get_secret.sh https://github.com/moolitayer/ocp-ansible-jenkins/blob/master/plugins/osh_get_secret.sh"
```
Comment (ilackarms) [inline deployer-test.sh]: why is it listed twice?

Comment (moolitayer) [reply deployer-test.sh]: It should support more then one URL. Here I arbitrarily tested with the same URL twice. Does not make sense in real life but ok for testing

Comment (ilackarms) [reply deployer-test.sh]: ahh sure

**Comment by moolitayer** [comment]
@simon3z @ilackarms can we merge?

@simon3z I'll add the ca script in another PR.

**Comment by moolitayer** [comment]
> @moolitayer @ilackarms it would be nice to have also the CA printed out so we'll make it easier to always use SSL

@cben which one is it?
```
$ ls /etc/origin/master/
admin.crt                                      master.etcd-client.key                         openshift-router.key
admin.key                                      master.kubelet-client.crt                      policy.json
admin.kubeconfig                               master.kubelet-client.key                      registry.crt
ca-bundle.crt                                  master.proxy-client.crt                        registry.key
ca.crt                                         master.proxy-client.key                        rh-cacert.crt
ca.key                                         master.server.crt                              scheduler.json
ca.serial.txt                                  master.server.key                              serviceaccounts.private.key
etcd.server.crt                                named_certificates/                            serviceaccounts.public.key
etcd.server.key                                openshift-master.crt                           service-signer.crt
master-config.yaml                             openshift-master.key                           service-signer.key
master-config.yaml.32974.2017-08-21@19:18:20~  openshift-master.kubeconfig                    session-secrets.yaml
master.etcd-client.crt                         openshift-router.crt                           
```

**Comment by cben** [comment]
/etc/origin/master/ca.crt

**Comment by moolitayer** [comment]
@simon3z ptal

**Comment by simon3z** [comment]
:+1:

**Comment by moolitayer** [comment]
Working in http://cm-jenkins.usersys.redhat.com/job/ocp-ansible-jenkins/122/console

**Pull request merged**